x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/data/BasicBlockTests.java

```diff
@@ -369,12 +373,12 @@ public void testConstantDoubleBlock() {
         }
     }
 
-    public void testBytesRefBlock() {
+    private void testBytesRefBlock(Supplier<BytesRef> byteArraySupplier, boolean chomp, org.mockito.ThrowingConsumer<BytesRef> assertions) {
```

Comment:
`CheckedConsumer`? I'm not used to seeing us reference the guts of mockito.

Reply:
Well, the difference between the existing UTF8 encoder test and the new WKB was tiny, and only needed this one extra assertion. This seemed the cleanest way to achieve that.